server/app/services/program/ProgramService.java

```diff
@@ -659,14 +659,22 @@ public ErrorAnd<ProgramDefinition, CiviFormError> updateLocalization(
             .setLocalizedConfirmationMessage(
                 programDefinition
                     .localizedConfirmationMessage()
-                    .updateTranslation(locale, localizationUpdate.localizedConfirmationMessage()))
-            .setStatusDefinitions(
-                programDefinition.statusDefinitions().setStatuses(toUpdateStatusesBuilder.build()))
-            .build()
-            .toProgram();
+                    .updateTranslation(locale, localizationUpdate.localizedConfirmationMessage()));
+    if (programDefinition.localizedSummaryImageDescription().isPresent()
```

Comment:
question/suggestion (blocking): I find this part a little hard to read. Is it possible to handle translations the same way that name, description, and confirmation message are? If not, I recommend factoring this logic out into a separate function.

Reply:
Great question. Like you'd mentioned in the other PR, the name and description are required program fields but the confirmation message is not. For confirmation message, we use `LocalizedStrings.of(DEFAULT_LOCALE, "")` to represent that we don't have any custom confirmation message. Since the message will always be an instance of `LocalizedStrings`, it's then easy to always call `localizedConfirmationMessage().updateTranslation(...)` here. For the program image description, I'm instead using `Optional.empty()` to represent that we don't have any description, which means that we have to check the optional everywhere and that's why this part is harder to read.

I think the question is:
1) Do we prefer the `LocalizedStrings.of(DEFAULT_LOCALE, "")` or `Optional.empty()` strategy for representing these non-required program fields?
- If we prefer `LocalizedStrings.of(DEFAULT_LOCALE, "")`, then I can update this PR and the previous PR to use that instead.
- If we prefer `Optional.empty()`, I can definitely pull this out into a helper method. And, that points to another question: should we go back and change the confirmation message implementation to also be `Optional.empty()`?

Reply:
Thank you so much for explaining! If an admin sets an image description and then goes back and clears it out, is it reset to `Optional.empty` or is it an empty string in that case? I think it's the former but want to make sure. Tbh I'm relatively new to using `Optional.empty` and I'm curious what you see as the tradeoffs. I like that it makes it super clear when we want to render the translations field, but it does seem to add some extra code.

Reply:
Yep it's reset to `Optional.empty()` (lines 903-905 of https://github.com/civiform/civiform/pull/5967/files#diff-33f37dc160bda78d0b594ff49d80c4ffacca5eefe12b4f1f37ef3c1f9105f546).

There are a bunch of different camps for when/how to use Optional, ranging from "never" to "only for return types but never for parameters or fields" to "frequently". https://nipafx.dev/inside-java-newscast-19/ and https://nipafx.dev/java-optional-opinions talk about the different camps and tradeoffs, and [this Reddit thread](https://www.reddit.com/r/java/comments/13rv1t2/why_exactly_does_brian_goetz_not_recommend_the/) has a lot of opinions :D Most of the discussion is around `null` vs `Optional`, whereas we're talking more about `Optional` vs "object whose internal fields are empty".

My 2 cents (and this really is just my opinion): I mostly agree with what's said in https://nipafx.dev/java-optional-opinions/#expressing-intent -- I think `Optional` is better than `null` and better than "object with empty fields" at expressing "this value is not set, and that's perfectly valid". For the image description in particular:
- I think the problem with null is that you need to hope that each caller remembers to check whether it's null, whereas `Optional` forces callers to check.
- I think using the object with empty fields is possibly misleading, because on first glance it looks like the field is always present, but certain configurations actually mean it's not present.

That being said, I do think it's easy to get carried away with using optionals too frequently, which makes the code too verbose and too difficult to understand (like you said), so we should be careful about where we use them. The image description felt like a good place to use them because the it is literally an optional field, but if we like using `null` or an empty `LocalizedStrings` better, than I'm happy to adjust!

Reply:
I added this as a specific implementation detail in https://github.com/civiform/civiform/issues/6007 if that looks good to you?